diff --color --unified --recursive --text --color old/ElmerGUI/Application/src/convergenceview.h new/ElmerGUI/Application/src/convergenceview.h
--- old/ElmerGUI/Application/src/convergenceview.h	2020-11-10 13:52:44.000000000 -0600
+++ new/ElmerGUI/Application/src/convergenceview.h	2021-10-15 13:38:06.034974088 -0500
@@ -51,8 +51,6 @@
 #include <qwt_plot_curve.h>
 #include <qwt_plot_grid.h>
 #include <qwt_legend.h>
-/*#include <qwt_data.h> <-- deprecated in Qwt6, using qwt_compat.h instead*/
-#include <qwt_compat.h>
 #include <qwt_text.h>
 #include <qwt_scale_engine.h>
 
@@ -76,8 +74,8 @@
   
 private:
   int d_count;
-  QwtArray<double> d_x;
-  QwtArray<double> d_y;
+  QVector<double> d_x;
+  QVector<double> d_y;
 };
 
 class Curve
