diff --unified --recursive --text --color old/ElmerGUI/Application/src/sifgenerator.cpp new/ElmerGUI/Application/src/sifgenerator.cpp
--- old/ElmerGUI/Application/src/sifgenerator.cpp	2020-11-10 13:52:44.000000000 -0600
+++ new/ElmerGUI/Application/src/sifgenerator.cpp	2021-04-22 15:58:53.093480110 -0500
@@ -177,16 +177,41 @@
 	     ui.simulationTypeCombo->currentText().trimmed());
   addSifLine("  Steady State Max Iterations = ",
 	     ui.steadyStateMaxIterEdit->text().trimmed());
-  addSifLine("  Output Intervals = ",
-	     ui.outputIntervalsEdit->text().trimmed());
+  
+  //Modify Output intervals size according to the number of values entered
+  QString qs = ui.outputIntervalsEdit->text().trimmed();
+  int sz = qs.count(' ') + 1:
+  if( sz > 1) {
+      addSifLine("  Output Intervals(" + QString::number(sz) + ") = ", qs);
+  }
+  else {
+      addSifLine("  Output Intervals = ", qs);
+  }
+  
   addSifLine("  Timestepping Method = ",
 	     ui.timesteppingMethodCombo->currentText().trimmed());
   addSifLine("  BDF Order = ",
 	     ui.bdfOrderCombo->currentText().trimmed());
-  addSifLine("  Timestep intervals = ",
-	     ui.timeStepIntervalsEdit->text().trimmed());
-  addSifLine("  Timestep Sizes = ",
-	     ui.timestepSizesEdit->text().trimmed());
+  
+  //Modify Timestep intervals size according to the number of values entered
+  qs = ui.timeStepIntervalsEdit->text().trimmed();
+  sz = qs.count(' ') + 1:
+  if( sz > 1) {
+      addSifLine("  Timestep intervals(" + QString::number(sz) + ") = ", qs);
+  }
+  else {
+      addSifLine("  Timestep intervals = ", qs);
+  }
+
+  //Modify TTimestep Sizes size according to the number of values entered
+  qs = ui.timestepSizesEdit->text().trimmed();
+  sz = qs.count(' ') + 1:
+  if( sz > 1) {
+      addSifLine("  Timestep Sizes(" + QString::number(sz) + ") = ", qs);
+  }
+  else {
+      addSifLine("  Timestep Sizes = ", qs);
+  }
 
   addSifLine("  Coordinate Scaling = ",
 	     ui.coordinateScalingEdit->text().trimmed());
@@ -198,7 +223,7 @@
   addSifLine("  Post File = ", 
 	     ui.postFileEdit->text().trimmed());
 
-  QString qs = ui.simulationFreeTextEdit->toPlainText();
+  qs = ui.simulationFreeTextEdit->toPlainText();
 
   if(!qs.isEmpty())
     te->append(qs);
